### DISCUSSION TITLE
Does it support SDNext?

### DISCUSSION THREAD

**Discussion by DivanoDova** [light-and-ray/sd-webui-replacer#77] [Q&A] [answered]
Wanted to try it but can't seems to works with SDNext.

Is it supported or will be in the future?

**Comment by light-and-ray** [comment]
No, I won't support sd-next

There is a fork for sd-next: https://github.com/RGX650/sd-webui-replacer-SDNEXT But it is very-very outdated. (Only fre days after I've made advanced options section) Maybe you can tag the author to ask them for updating

**Reply by RGX650** [reply]
@light-and-ray nice work !

**Comment by DivanoDova** [comment]
Oh ok thank you will try this.

@RGX650 Hi i tried you SDNext version of Replacer, it kinda works but seems outdated (it throw some errors but still give a output file). Did you plan to update it?

`    [Replacer]    Exception: plaintext_to_html() got an unexpected keyword argument 'classname'`

Thank you :)

**Comment by RGX650** [comment, marked as answer]
@DivanoDova Yes i plan to do it. however i am kind of busy with the BMAB extension at the moment. But yes ill come to it

**Reply by DivanoDova** [reply]
Oh thank you for the answer, i already fixed this error with my poor knowledge:
`[Replacer]    Exception: plaintext_to_html() got an unexpected keyword argument 'classname'`

Now i have a issue with the ModernUi of SDnext, the output preview is hidden with the new ui. But the images are in the output folder so not a big deal.
